src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -444,6 +447,26 @@ protected function getEntitiesByModule()
         }
         return $entitiesByModule;
     }
+    
+    /**
+     * Loads all modules and themes installed, including app. Uses the format of
+     * the @link ModuleLoader manifest for themes as well.
+     * Themes can be references with "themes/{theme name}".
+     */
+    private function getModulesAndThemes(): array
+    {
+        $modules = ModuleLoader::inst()->getManifest()->getModules();
+        // load themes as modules
+        $themes = array_diff(scandir(THEMES_PATH), ['..', '.']);
+        if ($themes) {
+            foreach ($themes as $theme) {
+                if (is_dir(THEMES_PATH . DIRECTORY_SEPARATOR . $theme)) {
+                    $modules[THEMES_DIR . '/' . $theme] = new Module(THEMES_PATH . DIRECTORY_SEPARATOR . $theme, BASE_PATH);
```

Comment:
I assume the `THEMES_DIR . '/' . $theme` part of this isn't intended to represent an actual file path at all, and is just a way to distinguish themes from modules in `getEntitiesByModule()`?
So to run text collector against a theme, you run `sake dev/tasks/i18nTextCollectorTask module=themes/my-theme-name`?

Reply:
correct, yes.

Reply:
In that case instead of using the `THEMES_DIR` constant just use the hardcoded string "themes" to avoid confusion

Reply:
done, thanks!